client/src/modules/accounts/edit/accounts.edit.js

```diff
@@ -196,58 +205,71 @@ function AccountEditController($rootScope, $state, AccountStore, Accounts,
     }
 
     if (vm.isCreateState) {
+      handleAccountCreateState();
+    } else {
+      handleAccountElseState();
```

Comment:
This isn't a very descriptive name.  If you aren't in the `CREATE` state, you must be in the `UPDATE` state. So a better name for this is `handleAccountUpdateState()`.